www/js/main.js

```diff
@@ -50,19 +40,59 @@ angular.module('emission.main', ['emission.main.diary', 'emission.main.common',
         controller: 'ControlCtrl'
       }
     }
+  })
+  .state('root.main.sensed', {
+    url: "/sensed",
+    views: {
+      'main-control': {
+        templateUrl: "templates/recent/sensedData.html",
+        controller: 'sensedDataCtrl'
+      }
+    }
+  })
+  .state('root.main.map', {
```

Comment:
Can you explain why we need to move the states in here? Now that recent.js is restored, the states are there. Why can't the buttons just navigate to 'root.main.recent.log' instead of 'root.main.log'?